src/js/paste.js

```diff
@@ -124,9 +125,10 @@ var PasteHandler;
 
                 if (!pastedHTML) {
                     pastedHTML = pastedPlain;
+                    onlyText = true;
```

Comment:
What if you didn't introduce this new variable `onlyText` and just removed this `if`?
On the next if below, `pastedHTML` will be empty and the condition will be false.